tcms_tenants/checks/tenants_env.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from django.core import checks
+
+
+@checks.register
```

Comment:
This isn't going to register the check automatically, see `apps.py` in https://github.com/kiwitcms/Kiwi/pull/2663